src/Deltas.proto

```diff
@@ -43,7 +44,7 @@ package Catalyst.Protocol.Delta;
 	bytes PreviousDeltaDfsHash = 2;
 	bytes MerkleRoot = 3;
 	bytes MerklePoda = 4;
-	uint32 TimeStamp = 5;
+	google.protobuf.Timestamp TimeStamp = 5;
```

Comment:
could this not be in a separate PR as timestamp changes aren't related to namespace naming convention changes?

Reply:
😂 sorry for the cognitive overload, I overestimated you 😂

Reply:
https://github.com/catalyst-network/protocol-protobuffs/pull/54